code/modules/mob/living/carbon/xenomorph/castes/ravager/castedatum_ravager.dm

```diff
@@ -46,7 +46,8 @@
 		/datum/action/xeno_action/activable/headbite,
 		/datum/action/xeno_action/activable/charge,
 		/datum/action/xeno_action/activable/ravage,
-		/datum/action/xeno_action/activable/ignore_pain,
+		/datum/action/xeno_action/activable/endure,
+		/datum/action/xeno_action/activable/rage,
```

Comment:
You're tracking these, but are you displaying them at round end?

Reply:
There's a lot of documented stats that aren't displayed; I'm just adding support. If you want me to display them I will